src/Apps/CollectorProfile/Routes/Saves2/Components/SavesArtworks.tsx

```diff
@@ -85,10 +92,13 @@ const QUERY = graphql`
     $page: Int
   ) {
     me {
-      collection(id: $collectionID) {
+      collection: collection(id: $collectionID) {
         ...SavesArtworks_collection @arguments(page: $page, sort: $sort)
       }
     }
+    defaultSaves: me {
+      ...SavesArtworks_me
+    }
```

Comment:
one question here: is that okay? I know that the nesting of `me` feels weird but wasn't sure how to spread it another way